testing/simapp/app.go

```diff
@@ -891,4 +892,14 @@ func (app *SimApp) setupUpgradeHandlers() {
 			ibcmock.ModuleName+icacontrollertypes.SubModuleName,
 		),
 	)
+
+	app.UpgradeKeeper.SetUpgradeHandler(
```

Comment:
[nit] not necessarily related to this PR, but the terminology of `Set` in `SetUpgradeHandler` is kind of confusing, in reality we are adding/appending a new upgrade handler. the name `AddUpgradeHandler` might makes more sense WDYT? We can create a separate issue if people agree with this.

cc @colin-axner @damiannolan

Reply:
this makes sense to me wrt re-naming!

Reply:
You'd have to change the [function](https://github.com/cosmos/cosmos-sdk/blob/main/x/upgrade/keeper/keeper.go#L70) on the SDK :smile: